apps/api/src/domain/models/context/commands/create-note-about-resource/note-about-resource-created.event.ts

```diff
@@ -0,0 +1,5 @@
+import { BaseEvent } from '../../../shared/events/base-event.entity';
+
+export class NoteAboutResourceCreated extends BaseEvent {
+    type = 'NOTE_ABOUT_RESOURCE_CREATED';
```

Comment:
I like the way it is worded now because it is consistent with other events, such as "SONG_CREATED". They all have the form "N_CREATED", where in is a noun. That this is a noun phrase seems like a fine generalization.